django/thunderstore/repository/tasks.py

```diff
@@ -46,3 +47,11 @@ def delete_user(user_id: int) -> None:
             comment.author = ghost_user
             comment.save(update_fields=("author",))
         user.delete()
+
+
+@shared_task
+def clean_up_comments() -> None:
+    """Deletes comments with a deleted parent."""
+    for comment in Comment.objects.iterator():
+        if comment.thread is None:
```

Comment:
`Comment.objects.filter(thread=None)` is not allowed but this feels very inefficient